src/app/components/SkipToContentWrapper.tsx

```diff
@@ -19,7 +20,8 @@ export default class SkipToContentWrapper extends Component {
   private scrollToTarget = (event: React.MouseEvent<HTMLAnchorElement>) => {
     if (this.mainContent) {
       event.preventDefault();
-      scrollTo(this.mainContent);
+      scrollToContent(this.mainContent);
+      (this.mainContent as HTMLElement).focus();
```

Comment:
you shouldn't need this `as HTMLElement`, its already a `HTMLDivElement`, if you're getting the wrong `HTMLDivElement` you might have to import it from `lib.dom`

Reply:
I think it was complaining that HTMLDivElement did not respond to focus... I'll try the import and see what happens.

Reply:
If I add this import here, I also have to add it to SkipToContent.ts but then the mergeRefs function gets a type error because this HTMLDivElement is not assignable to the non-OpenStax HTMLElement and I'm not sure how to fix that one...